client/src/themes/theme.ts

```diff
@@ -21,17 +21,22 @@ export const theme = createMuiTheme({
     h3: {
       fontSize: 18,
       color: '#000000',
-      fontWeight: 500,
-      fontFamily: "'Arial'",
+      fontWeight: 300,
+      fontFamily: "'Roboto'",
```

Comment:
It would be ideal to modify the font family for the entire Typography object in the theme (e.g. `theme.typography.fontFamily`) in order to provide a consistent UI/UX throughout the application 👍🏼 You can explore the theme object in more detail [here](https://v4.mui.com/customization/default-theme/#explore).